js/components/agency_preview.jsx

```diff
@@ -0,0 +1,72 @@
+/*
+ * AgencyPreview
+ *
+ * Displays information for a decentralized agency.
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import { List } from 'immutable';
+
+import { Agency } from '../models';
+
+function AgencyPreview({ agency, agencyComponentsForAgency, onAgencySelect }) {
+  const description = agency.mission();
+  return (
+    <div className="agency-preview usa-grid-full">
+      <div className="usa-width-one-whole">
+        <h3>{agency.name}</h3>
+      </div>
+      <div className="usa-width-one-half">
+        <div className="heading-2">
```

Comment:
To be totally semantic, this should be either an `h4` or a `p` tag with a class. I'd lean toward the latter, called something like `agency-preview_list-hed` (rather than mixing in atomic-style selectors). Then that can just `@extend` the `h2` rules.

Reply:
👍 should I remove the atomic `heading-2` scss?